server/src/main/java/no/fractal/socket/send/MessageDispatcher.java

```diff
@@ -0,0 +1,47 @@
+package no.fractal.socket.send;
+
+import no.fractal.util.StreamUtils;
+
+import java.io.BufferedOutputStream;
+import java.lang.annotation.Target;
+import java.util.Vector;
+
+public class MessageDispatcher {
+
+    private Vector<AbstractMessage> messageQue;
```

Comment:
messageQueue

Reply:
Que is defined as the abbreviation for Quebec and the Spanish word for what.

man lærer så lenge man lever